Storybook & Tailwind Issue

```
ERROR in ./node_modules/tailwindcss/tailwind.css (./node_modules/css-loader/dist/cjs.js??ref--7-1!./node_modules/@storybook/core/node_modules/postcss-loader/src??ref--7-2!./node_modules/postcss-loader/dist/cjs.js??ref--10-0!./node_modules/tailwindcss/tailwind.css)
Module not found: Error: Can't resolve './public/assets/backgrounds/home.jpg' in '/Users/jamesyaputra/Workspace/PINUS/pinus-client/node_modules/tailwindcss'
```

Error in development environment; issue with how webpack and/or tailwind resolves paths found in `tailwind.config.css`

--- a/src/pages/index.tsx
+++ b/src/pages/index.tsx
@@ -64,7 +64,7 @@ const Home: React.FC = () => {
           </div>
           <div className={`col-start-6 col-span-6 row-start-1 row-span-5`}>
             <Carousel
-              slides={[{ title: "", image: "./assets/images/nuansa.jpg" }]}
+              slides={[{ title: "", image: "/assets/images/nuansa.jpg" }]}
             />
           </div>
           <div className={`col-start-6 col-span-7 row-start-1 row-span-6`}>

--- a/tailwind.config.js
+++ b/tailwind.config.js
@@ -15,10 +15,10 @@ module.exports = {
       secondary: "#EDEDED",
     }),
     backgroundImage: () => ({
-      home: "url('public/assets/backgrounds/home.jpg')",
-      "home-showcase1": "url('public/assets/images/home-1.jpg')",
-      "home-showcase2": "url('public/assets/images/home-2.jpg')",
-      "home-showcase3": "url('public/assets/images/home-3.jpg')",
+      home: "url('/assets/backgrounds/home.jpg')",
+      "home-showcase1": "url('/assets/images/home-1.jpg')",
+      "home-showcase2": "url('/assets/images/home-2.jpg')",
+      "home-showcase3": "url('/assets/images/home-3.jpg')",
     }),
     screens: {
       sm: { max: "640px" },

